generators/app/templates/graphQL/README.md

```diff
@@ -0,0 +1,149 @@
+# graphql
```

Comment:
mmm, parts of the readme are equal and parts are different comparing to the nodejs one. What do you think about this?

Reply:
ok!